### DISCUSSION TITLE
Vote on spectrum class name in Specutils 2.0

### DISCUSSION THREAD

**Discussion by rosteen** [astropy/specutils#1087] [Polls]
For additional context, see #1056. Github has the capability to run actual polls now, so I figured this would be a more organized way to let people vote on the proposed options. I hope that people can vote for multiple options, but I'm not sure as this is the first time I'm using this GitHub feature. I'll leave this open for a few weeks (no real rush, but I hope to get specutils 2.0 out in <6 months).

**Comment by pllim** [comment]
While I like `WigglyLineDataFromATelescopeOrSomeOtherInstrumentIGuessWeDontDiscriminate`, maybe we should shorten it to `WLDFATOSOIIGWDD`.

**Reply by keflavich** [reply]
I would like to store all of my `WLDFATOSOIIGWDD` metadata in `fits` `hdus`, with my `gcws` in `asdf`.  Seems well-matched!

**Reply by hamogu** [reply]
That's what you say. I want my `gwcs` in `fits hdus` and my metadata in `asdf`, while I prefer the actual spectrum in in Excel spreadsheets with font set to wingdings.

**Reply by eteq** [reply]
```
class WLDFATOSOIIGWDD:
    def __init__(self, *args, **kwargs):
        super(self).this.display_font = ('wingdings', 'comic sans')
        self.file.close()
        ... all the rest is unimportant ...   
```

**Comment by rosteen** [comment]
`Spectrum` was the clear winner here, closing this. I'll open a PR soonish to the 2.0 dev branch to do the renaming and add a deprecation warning to the old class name.
